api/hypershift/v1beta1/gcpprivateserviceconnect_types.go

```diff
@@ -38,12 +38,20 @@ const (
 
 // GCPPrivateServiceConnectSpec defines the desired state of PSC infrastructure
 type GCPPrivateServiceConnectSpec struct {
-	// forwardingRuleName is the name of the Internal Load Balancer forwarding rule
+	// loadBalancerIP is the IP address of the Internal Load Balancer
+	// Populated by the observer from service status
 	// +required
-	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MinLength=7
+	// +kubebuilder:validation:MaxLength=15
+	// +kubebuilder:validation:Pattern=`^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$`
```

Comment:
could this ever be ipv6?

Reply:
Thanks @csrwng , I was trying to avoid a complex regular expression. 

I've updated it.